ace/validator.go

```diff
@@ -303,18 +303,15 @@ func validateContainerAnnotations(metadataURL string, pm *schema.PodManifest) re
 	return r
 }
 
-func validateContainerMetadata(metadataURL string, pm *schema.PodManifest) results {
+func validatePodMetadata(metadataURL string, pm *schema.PodManifest) results {
 	r := results{}
 
-	uid, err := metadataGet(metadataURL, "/pod/uid")
+	_, err := metadataGet(metadataURL, "/pod/uuid")
```

Comment:
I can't now. Since UUID is not part of manifest there's no other way to know uuid